### PR TITLE
Add Apache 2.0 license

### PR DESCRIPTION
**Pull request by jar349**
This pull request already has the changes from @shlomisut merged.  It adds a LICENSE file whose contents are the Apache 2.0 license.

Note that accepting either of our pull requests will change the version of the project to 0.7.1.2 because it is our second update that tracks version 0.7.1 of Dropwizard.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -5,7 +5,7 @@ apply plugin: 'spring-boot'
 apply plugin: 'groovy'
 apply plugin: 'signing'
 
-version = '2.0.1'
+version = '0.7.1.2'
 group = 'com.bericotech'
```
Comment (shlomisut) [inline build.gradle:9]: I wonder about the group...

Comment (rclayton) [inline build.gradle:9]: Agreed.  We need to figure out the group.  I own a couple of domains, including "prostack.io".  That's one possibility.  However, I'm not inclined to buy one specific to Fallwizard!

Comment (shlomisut) [inline build.gradle:9]: I'm actually not too familiar with nexus requirement, must it have an actual website behind it?

**Comment by jar349** [comment]
If we setup a github.io webpage for the fallwizard org, then we can use io.github.fallwizard since fallwizard.github.io would point to that website.

Or we can do what junit does and just make the groupId be Fallwizard

Finally, though I like it less, I own johnruiz.com and could use that.

Personally, my favorite is #1.

File: build.gradle (hunk 1 above)
Comment (rclayton) [inline build.gradle:9]: @shlomisut - Sonatype requires us to have a unique GroupID, generally for a domain that we own (unless as John points out, the GroupID can't possibly be a valid internet domain).

**Comment by rclayton** [comment]
@jar349 I was thinking something like your first option, but for some reason I had a "derp" moment and thought it would be unwise to use the github.com domain name.  Now I realize how dumb I was being since the "fallwizard" subdomain would be tacked on.  Let's do that.

**Comment by rclayton** [comment]
I'm going to reject the pull request only because the group id needs to change.

**Comment by rclayton** [comment]
Ok, per conversation with John, he will update group ID after merge.

**Comment by jar349** [comment]
I just went ahead and pushed the change.  Seemed silly to create a whole pull request.

**Comment by shlomisut** [comment]
:+1:

**Pull request merged**